addons/mass_mailing/models/mailing.py

```diff
@@ -754,6 +757,36 @@ def _prepare_statistics_email_values(self):
     # TOOLS
     # ------------------------------------------------------
 
+    def _get_mailing_name(self, base_name, subject):
+        """Generate a name for the mailing.
+
+        :param base_name: base name to use
+        :param subject: subject of the mailing
+        """
+        base_name = base_name or ''
+        if subject and not base_name:
+            base_name = _(
+                "%s (Mailing created on %s)",
+                subject,
+                fields.Date.to_string(fields.Datetime.now().date()),
```

Comment:
(Small note: I suggested "fields.Date.to_string(**fields.Date.today()**)" in the previous PR, does it not work?)

Reply:
Yes it also work :) Let keep the simplest version :)